### DISCUSSION TITLE
Atera to Budibase: API Query Customers

### DISCUSSION THREAD

**Discussion by ctroche28** [Budibase/budibase#6933] [Help] [answered]
Hi,

I am new to all this and am trying to query -get all customers from Atera. First, I imported the query from a curl. It imports the information successfully. However, the response body shows similar to this...not adding in the actual customer data:

{
  "items": [
    {
      "CustomerID": 27,
      "CustomerName": "Example",
      "CreatedOn": "2022-07-01T19:35:29Z",
      "LastModified": "2022-07-01T19:35:29Z",
      "BusinessNumber": null,
      "Domain": "",
      "Address": null,
      "City": null,
      "State": null,
      "Country": null,
      "Phone": null,
      "Fax": null,
      "Notes": null,
      "Logo": null,
      "Links": null,
      "Longitude": 0,
      "Latitude": 0,
      "ZipCodeStr": null
    },
{
      "CustomerID": 26,
      "CustomerName": "Example",
      "CreatedOn": "2022-07-01T19:35:29Z",
      "LastModified": "2022-07-01T19:35:29Z",
      "BusinessNumber": null,
      "Domain": "",
      "Address": null,
      "City": null,
      "State": null,
      "Country": null,
      "Phone": null,
      "Fax": null,
      "Notes": null,
      "Logo": null,
      "Links": null,
      "Longitude": 0,
      "Latitude": 0,
      "ZipCodeStr": null
    },
  ],
  "totalItemCount": 0,
  "page": 0,
  "itemsInPage": 0,
  "totalPages": 0,
  "prevLink": "string",
  "nextLink": "string"
}
So, when you review the preview, it displays like this:
![image](https://user-images.githubusercontent.com/110132666/181375639-ae199277-35dc-4394-97d7-6d8fca42b42a.png)

Is there a way to exclude/remove the "items" text or display the array so that the table displays all customers?

**Comment by melohagan** [comment, marked as answer]
Hey @ctroche28 

You'll need to transform your response: https://docs.budibase.com/docs/transformers

So in your case the transformer would look something like:

```js
return data.items
```

**Reply by ctroche28** [reply]
This worked! Thank you so much!

**Reply by melohagan** [reply]
You're welcome!
